usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
```

Comment:
I would also check here that the agent actually runs. Maybe move those checks from the start script here?